frontend/src/metabase/ui/components/inputs/TextInputBlurChange/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { TextInput } from "@mantine/core";
+import type { TextInputProps } from "@mantine/core";
+import _ from "underscore";
+import type { ChangeEvent } from "react";
+import { useCallback, useLayoutEffect, useRef, useState } from "react";
+import { useUnmount } from "react-use";
+
+type Value = string | undefined;
+
+export interface TextInputBlurChangeProps
+  extends Omit<TextInputProps, "value" | "onBlur" | "ref"> {
+  value: Value;
+  onBlurChange: (event: { target: HTMLInputElement }) => void;
+  normalize?: (value: Value) => Value;
+}
+
+/**
+ * A wrapper around TextInput to be used with onBlurChange prop.
+ *
+ * In case you don't need it, use TextInput directly.
+ */
+export function TextInputBlurChange({
```

Comment:
There's 1 usage of `InputBlurChange` left, can we replace it with `TextInputBlurChange` now?

Reply:
Got it, thanks for checking! :pray: